packages/core/src/period-picker/PeriodPicker.component.js

```diff
@@ -178,6 +181,18 @@ class PeriodPicker extends React.Component {
         return this.renderOptionPicker('week', weeks);
     }
 
+    renderBiWeekPicker() {
```

Comment:
There are a lot of complexities related to calendar time, so some of the assumptions here do not hold up: e.g. some years have 53 weeks. Every 28 years there is 54 weeks.

We should introduce Moment JS to deal with the complexity related to time and date.